tests/cpp/test_tutorial.cpp

```diff
@@ -670,24 +672,25 @@ TEST_F(Tutorial, IdModelReshapeAnalysis) {
 
   // Now, suppose we can say the inputs are exactly mapped. We
   // can manually add mappings:
-  for (const auto i : c10::irange(tv0->getRootDomain().size())) {
-    exact_graph.mapVals(tv0->getRootDomain().at(i), tv1->getRootDomain().at(i));
+  for (const auto i : c10::irange(tv0->getMaybeRootDomain().size())) {
+    exact_graph.mapVals(
+        tv0->getMaybeRootDomain().at(i), tv1->getMaybeRootDomain().at(i));
   }
 
   // Now, tv2 and tv3 should be fully mapped, including their root,
   // intermediate and leaf domains.
 
   // Check the root domains.
-  for (const auto i : c10::irange(tv2->getRootDomain().size())) {
+  for (const auto i : c10::irange(tv2->getMaybeRootDomain().size())) {
```

Comment:
I think here `getRootDomain` makes more sense. We could also check the rfactor domain, but all the intermediate domains are already indeed checked below.

Reply:
fixed